dev/build-fleet

```diff
@@ -18,6 +18,14 @@ fi
 
 export GOOS=linux
 
+# The name of the container image created here is a potential source of conflict
+# when fleet is set up simultaneously, as several scripts are calling
+# build-fleet and it always creates a container image with the same name
+# (`rancher/fleet`, rancher/fleet-agent`) and tag (`dev`). Conflicts can simply
+# be avoided by making sure this script is not called sequentially, e.g. by
+# creating multiple clusters on a single host sequentially rather than
+# simultaneously.
```

Comment:
I'm not sure what calling a script "sequentially" means.
What would you think of something like this? How does it compare to what you have/had in mind?
```suggestion
# be avoided by making sure that images `rancher/fleet:dev` and
# `rancher/fleet-agent:dev` are rebuilt from the right context (eg. git clone,
# checkout or worktree) before being imported into clusters created for testing
# that context.
# This can be easily achieved by ensuring that, for instance, at most one instance
# of `./dev/setup-single-cluster` or `./dev/setup-multi-cluster` runs at any given
# point in time.
```